pkg/controller/certificates/internal/secretsmanager/secret.go

```diff
@@ -236,5 +236,25 @@ func (s *SecretsManager) setValues(crt *cmapi.Certificate, secret *corev1.Secret
 		secret.Annotations[cmapi.URISANAnnotationKey] = strings.Join(utilpki.URLsToString(x509Cert.URIs), ",")
 	}
 
+	// copy annotations from SecretTemplate. Do not overwrite if exists.
```

Comment:
Not overwriting if it already exists will also mean we can't _update_ these if it is changed on the Certificate. Is that desirable?

Reply:
Indeed not desirable. I've added this check to protect existing annotations set by cert-manager) such cmapi.URISANAnnotationKey) of being overwritten.

Reply:
fixed. Now updated keys are being copied.